src/docs/testing/ui-performance/index.md

```diff
@@ -402,8 +379,7 @@ _Because raster cache entries are expensive to construct and take up loads
 of GPU memory, cache images only where absolutely necessary._
 
 You can see which images are being cached by enabling the
-[PerformanceOverlayLayer.checkerboardRasterCacheImages]({{site.api}}/flutter/rendering/PerformanceOverlayLayer/checkerboardRasterCacheImages.html)
-switch.
+[PerformanceOverlayLayer.checkerboardRasterCacheImages][] switch.
```

Comment:
Do we explain to users how to enable these settings? This requires a power user who is comfortable editing code in package:flutter
If these flags are useful enough to call out in the doc, we should add buttons to the IDEs to toggle them on and off. @kenzieschmoll lets add ext.flutter.* service extensions for each of these.

Reply:
OK, leaving this as-is for now. Let me know if you enable additional UI.